### ISSUE TITLE
The optimal range of the ’alpha‘ parameter in the TAU model.

### ISSUE THREAD

**Issue by QUSTwh** [chengtan9907/OpenSTL#201]
When using the TAU model, what value should the 'alpha' parameter in the LOSS function be set to? 
What impact does the size of this parameter have?
`loss = self.criterion(pred_y, batch_y) + self.hparams.alpha * self.diff_div_reg(pred_y, batch_y)`
